backend/src/test/map.rs

```diff
@@ -13,11 +13,11 @@ use diesel::ExpressionMethods;
 use diesel_async::{scoped_futures::ScopedFutureExt, RunQueryDsl};
 
 #[actix_rt::test]
-async fn test_can_find_map() {
+async fn test_can_search_maps() {
```

Comment:
Test case is quite long. Can you refactor a bit to have common place somewhere else and then have a few short test cases? (More important for plant searching, though).

Reply:
I don't think that is really possible. The `test_can_search_maps` executes two different requests, but both of them should in my opinion be part of this test case.